### PR TITLE
Implements game status for a player distance

### PR DESCRIPTION
**Pull request by rarellano**

### CODE REVIEW

File: game/views.py [hunk 1]
```diff
@@ -45,3 +53,31 @@ def get_queryset(self):
...
+        self.player_content = Content.objects.get(content_type=ContentType.objects.get(pk=17),
+                                                     content_id=self.player.pk)
+        game = Game.objects.get(pk=pk)
+        contents = Content.objects.filter(game=game)
+
+        q = Q(position__distance_lte=(self.player_content.position, D(m=game.preferences.vision_distance)))
+
+        result = {
+                    'npcs': [ContentSerializer(x).data
+                             for x in contents.filter(content_type=ContentType.objects.get(pk=16))
```
Comment (Virako) [inline game/views.py]: Not use magic numbers, get ContentType by name.
Maybe ContentType with pk in each machine are differents.

File: game/views.py [hunk 2]
```diff
@@ -45,3 +53,31 @@ def get_queryset(self):
...
         return games
+
+
+class GameStatusForPlayer(views.APIView):
+    permission_classes = (IsAuthenticatedOrReadOnly,)
+
+    def get(self, request, version, pk, *args, **kwargs):
+        self.player = Player.objects.get(user=request.user)
+        self.player_content = Content.objects.get(content_type=ContentType.objects.get(pk=17),
+                                                     content_id=self.player.pk)
```
Comment (Virako) [inline game/views.py]: Style guide: C0330: Wrong continued indentation

File: game/views.py [hunk 3]
```diff
@@ -45,3 +53,31 @@ def get_queryset(self):
...
+        game = Game.objects.get(pk=pk)
+        contents = Content.objects.filter(game=game)
+
+        q = Q(position__distance_lte=(self.player_content.position, D(m=game.preferences.vision_distance)))
```
Comment (Virako) [inline game/views.py]: Style guide:
* C0301: lines too long in several lines 
* C0330: Wrong continued indentation in several lines

**Comment by Virako** [comment]
The PR has some style errors and the tests are neccessary

**Review by Virako — APPROVED** [review]

File: game/views.py (hunk 1 above)
Comment (rarellano) [reply game/views.py]: Done

File: game/views.py (hunk 3 above)
Comment (rarellano) [reply game/views.py]: Done

File: game/views.py (hunk 2 above)
Comment (rarellano) [reply game/views.py]: Done

**Comment by rarellano** [comment]
Now exist test for this endpoint

**Review by danigm — APPROVED** [review]

**Pull request merged**